addons/website_sale/controllers/main.py

```diff
@@ -315,12 +315,16 @@ def product(self, product, category='', search='', **kwargs):
         def compute_currency(price):
             return product.currency_id._convert(price, pricelist.currency_id, product._get_current_company(pricelist=pricelist, website=request.website), fields.Date.today())
 
+        domain = self._get_search_domain(search, category, attrib_values)
+        product_count = request.env['product.template'].search_count(domain)
+
         if not product_context.get('pricelist'):
             product_context['pricelist'] = pricelist.id
             product = product.with_context(product_context)
 
         values = {
             'search': search,
+            'search_count': product_count,
```

Comment:
Where is this used ? "fix fond in product" ?

Reply:
It is the found count in product page. When you perform a research with the "Show # found" customization enabled. I Will update the commit message accordingly.

Reply:
JKE decided as I explained to you: make the xml change to not show the number of found on a product page